lib/src/main/java/graphql/nadel/ServiceExecutionFactory.kt

```diff
@@ -13,15 +9,4 @@ interface ServiceExecutionFactory {
      * @return a function that can be called to get data from that service
      */
     fun getServiceExecution(serviceName: String): ServiceExecution
-
-    /**
-     * todo: why does this belong here and not in the Nadel constructor?
-     *
-     * Called to return underlying type definitions for a schema
-     *
-     * @param serviceName the name of the service
-     *
-     * @return a [graphql.schema.idl.TypeDefinitionRegistry] of all the underlying schema types
-     */
-    fun getUnderlyingTypeDefinitions(serviceName: String): TypeDefinitionRegistry
```

Comment:
Moved this to `Nadel.Builder.underlyingSchema` etc.

Putting this here would suggest that we dynamically get the schema e.g. via introspection. Which is an interesting idea, but for now we're _not_ doing it. So I'm moving it closer to the actual usage.